tests/pages/compareSystemParametersPage.js

```diff
@@ -17,9 +17,14 @@ module.exports = {
     },
 
     verifyMetricsExistence () {
-        for (var i in this.metrics) {
+        I.click("/html/body/grafana-app/div/div/div/react-container/div/div[2]/div/div[1]/div/div[1]/dashboard-submenu/div/div[5]/dash-links-container/dash-link[2]/div/a");
```

Comment:
Can you please check your PR? I don't think you merged the changes I pushed, please update your tests accordingly